cypress/page/student-registration-form.page.ts

```diff
@@ -24,19 +22,13 @@ class StudentRegistrationForm {
     this.emailText = '#userEmail';
     this.genderBtn = '[type="radio"]';
     this.mobileNumberText = '#userNumber';
-    this.hobbiesCheck = '[type="checkbox"]';
+    this.hobbiesCheck = '#hobbiesWrapper input+label';
     this.currentAddressText = '#currentAddress';
     this.submitBtn = '#submit';
     this.stateList = '#state > div > div.css-1hwfws3 > div.css-1wa3eu0-placeholder';
     this.cityList = '#city > div > div.css-1hwfws3 > div.css-1wa3eu0-placeholder';
-    this.NCROstate = "#react-select-3-option-0";
-    this.delhiCity = "#react-select-4-option-0";
-    this.hobbies = "Sports, Reading, Music";
-    this.myhashhobbies = new Map<string, string>([
-      ["1", "Sports"],
-      ["2", "Reading"],
-      ["3", "Music"]
-    ]);
+    this.NCROstate = "#react-select-3-input";
+    this.delhiCity = "#react-select-4-input";
```

Comment:
why not name this as `delhiCity` or something similar

Reply:
resolved